packages/ibm-products/src/components/index.ts

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright IBM Corp. 2024, 2024
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+import '../feature-flags';
+
+export * from './AboutModal';
+export * from './APIKeyModal';
+export * from './Cascade';
+export * from './ComboButton';
+export * from './CreateFullPage';
+export * from './CreateModal';
+export * from './CreateSidePanel';
+export * from './CreateTearsheet';
+export * from './CreateTearsheetNarrow';
+export * from './EmptyStates';
+export * from './ExportModal';
+export * from './ExpressiveCard';
+export * from './HTTPErrors';
+export * from './ImportModal';
+export * from './MultiAddSelect';
+export * from './NotificationsPanel';
+export * from './PageHeader';
+export * from './ProductiveCard';
+export * from './RemoveModal';
+export * from './Saving';
+export * from './SidePanel';
+export * from './SingleAddSelect';
+export * from './StatusIcon';
+export * from './TagSet';
+export * from './Tearsheet';
+export * from './Toolbar';
+export * from './UserProfileImage';
+export * from './WebTerminal';
+export * from './EditSidePanel';
+export * from './OptionsTile';
+export * from './EditInPlace';
+export * from './DataSpreadsheet';
+export * from './Datagrid';
+export * from './EditTearsheet';
+export * from './EditTearsheetNarrow';
+export * from './EditFullPage';
+export * from './EditUpdateCards';
+export * from './Checklist';
+export * from './Coachmark';
+export * from './CoachmarkBeacon';
+export * from './CoachmarkButton';
+export * from './CoachmarkFixed';
+export * from './CoachmarkOverlayElement';
+export * from './CoachmarkOverlayElements';
+export * from './CoachmarkStack';
+export * from './Guidebanner';
+export * from './InlineTip';
+export * from './NonLinearReading';
+export * from './BigNumbers';
+export * from './TruncatedList';
+export * from './InterstitialScreen';
+export * from './InterstitialScreenView';
+export * from './InterstitialScreenViewModule';
+export * from './DelimitedList';
+export * from './Decorator';
+export * from './DecoratorLink';
+export * from './DecoratorSingleButton';
+export * from './DecoratorDualButton';
+export * from './DescriptionList';
+export * from './FullPageError';
+export * from './SearchBar';
+export * from './Nav';
```

Comment:
There are some [ci issues](https://github.com/carbon-design-system/ibm-products/actions/runs/10321685255/job/28575148442?pr=5807).

If we look at this export for example, this will export everything from `Nav/index.ts`. Previously we only exported `Nav` but now `NavItem` and `NavList` are also exported. This is what we want (and is actually a bug previously as these should be exported), but it also means those components need to be added `package-settings.js` and also need to include the `pkg.checkComponentEnabled` check in the component.

Reply:
I see, so NavList and NavItem should be exported?

Reply:
I exported them in the latest changes